salt/states/boto_ec2.py

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
+'''
+Manage EC2
+
+.. versionadded:: TBD
```

Comment:
Actually that one should be Beryllium. The current develop branch will be the Beryllium release. :)